src/containers/login/index.js

```diff
@@ -23,9 +23,9 @@ const Login = props => (
                 <form>
                     <MuiThemeProvider>
                         <div class="textField">
-                            <TextField placeholder="Email" class="field" /><br /><br />
+                            <TextField placeholder="Email" class="field" style={{ width: 400 }} /><br /><br />

```

Comment:
Inline style can be embedded then br tags can be removed with the style defining it.

Reply:
Ok, well I found another way of specifying the width of a material-ui textField, so I am just going to do it like that, it will then work without the <br /> tags.